### PR TITLE
#2 add retry annotation

### PR DESCRIPTION
**Pull request by Emily-Jiang**
Signed-off-by: Emily Jiang <emijiang6@googlemail.com>

### CODE REVIEW

**Comment by Emily-Jiang** [comment]
Please review as I would like to merge in soon.

File: api/src/main/java/org/eclipse/microprofile/fault/tolerance/cdi/Retry.java [hunk 1]
```diff
@@ -0,0 +1,107 @@
+/*
+ * Copyright (c) 2016-2017 Contributors to the Eclipse Foundation
```
Comment (johnament) [inline api/src/main/java/org/eclipse/microprofile/fault/tolerance/cdi/Retry.java]: Wrong year.

File: api/src/main/java/org/eclipse/microprofile/fault/tolerance/cdi/Retry.java [hunk 2]
```diff
@@ -0,0 +1,107 @@
...
+     *
+     * @return Specify the failure to abort on
+     */
+    Class<? extends Throwable>[] abortOn() default { Throwable.class };
+
+    /**
+     *
+     * @return The fallback method name
+     */
+    String fallBack();
```
Comment (johnament) [inline api/src/main/java/org/eclipse/microprofile/fault/tolerance/cdi/Retry.java]: Fallback is its own annotation.

**Comment by Emily-Jiang** [comment]
updated the changes and merged in just this one.

**Pull request closed without merging**